modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -56,9 +56,19 @@ apiVersion: "k8s.cni.cncf.io/v1"
 ----
 <1> The type is `bond`.
 <2> The `ifname` attribute specifies the name of the bond interface.
-<3> The `failover` attribute is mandatory for active-backup mode.
-<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
-<5> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+<3> The `mode` attribute specifies the bonding mode. 
++
+[NOTE]
+====
+The bonding modes supported are:
+
+* 0 - balance-rr
```

Comment:
Will you please confirm that you need these numbers? The sample YAML uses the mode name. If you only need the mode names, you could present them without the note. In either case, the literal values need to be enclosed in backticks.